qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/0-cluster/start.yaml

```diff
@@ -1,8 +1,8 @@
 meta:
 - desc: |
-   Insatll and run ceph on one node,
+   Install and run ceph on one node,
    with a separate client 1.
-   Upgrade client 1 to octopus
+   Upgrade client 1 to quincy
```

Comment:
done

Reply:
No, it's not done.  This is in the `client-upgrade-octopus-pacific` directory, so `client.1` is upgraded to `pacific`, not `quincy`.  I suggested removing this change from this PR and opening a separate PR for `client-upgrade-octopus-pacific` with not just typo fixes but also the removal of `2-features` directory precisely to avoid pacific vs quincy confusion.

Reply:
AFAIK in the `client-upgrade-octopus-pacific directory`, `client.1` is to be upgraded to pacific because it's used for `pacific`

I think it's unrelated to this PR thou
Pls resolve of agree

Reply:
Yes, it would be upgraded to pacific -- that is exactly what I'm saying.  However you changed it to say quincy here:
```
diff --git a/qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/0-cluster/start.yaml b/qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/0-cluster/start.yaml
index 8354b1772bdd..a0f265cf5034 100644
--- a/qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/0-cluster/start.yaml
+++ b/qa/suites/upgrade-clients/client-upgrade-octopus-pacific/octopus-client-x/rbd/0-cluster/start.yaml
@@ -1,8 +1,8 @@
 meta:
 - desc: |
-   Insatll and run ceph on one node,
+   Install and run ceph on one node,
    with a separate client 1.
-   Upgrade client 1 to octopus
+   Upgrade client 1 to quincy
    Run tests against old cluster
 roles:
 - - mon.a
```